src/profile/helpers/editData.ts

```diff
@@ -115,11 +118,15 @@ function isSaving(allItems: EditData[]): boolean {
   return !!allItems.find(item => !!item.saving);
 }
 
-function getNewItem(allItems: EditData[]): EditData | undefined {
-  return allItems.find(item => item.id === '');
+export function isNewItem(data: EditData): boolean {
+  return data.id === '';
 }
 
-function hasNewItem(allItems: EditData[]): boolean {
+export function getNewItem(allItems: EditData[]): EditData | undefined {
```

Comment:
Is there always only 1 new item in the array? Could it be more safe to rename this: getFirstNewItem?

Reply:
There can be only one (or none), so no need to clarify it.